internal/mode/static/manager.go

```diff
@@ -200,10 +200,20 @@ func StartManager(cfg config.Config) error {
 
 		ngxruntimeCollector = collectors.NewManagerMetricsCollector(constLabels)
 		handlerCollector = collectors.NewControllerCollector(constLabels)
+
+		ngxruntimeCollector, ok := ngxruntimeCollector.(prometheus.Collector)
+		if !ok {
+			return fmt.Errorf("ngxruntimeCollector is not a prometheus.Collector: %w", status.ErrFailedAssert)
```

Comment:
missing unit tests

Reply:
This is just adding an error check, I don't think there's any unit tests that I can add?